resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -31,60 +22,74 @@ type DmEntryCreateFormPropType = {
     editId?: number
     user_id?: number
     adventures: adventureType[]
+    characters: CharacterData[]
 }
 
 type DmEntryFormDataType = {
     adventure_id: number
     length: number
-    location: string
+    levels: number
+    gp: number
+    location: string | null
     date_played: string | null
+    choice: string | null
+    character_id: number | null
     notes: string
     items: ItemData[]
     type: string
     user_id: number | null | undefined
 }
 
-const StyledBox = styled(Box)`
-    padding: 32px 0px 16px 0px;
-`
-
-const FormBox = styled(Box)`
-    flexgrow: 1;
-    margin-top: 16px;
-`
+const choices = [
+    {
+        value: 'advancement',
+        text: 'Advancement',
+    },
+    {
+        value: 'magic_item',
+        text: 'Magic Item',
+    },
+    {
+        value: 'campaign_reward',
+        text: 'Campaign Reward',
+    },
+]
 
 const StyledGrid = styled(Grid)`
     align-items: center;
     justify-content: center;
     margin-bottom: 16px;
 `
-const StyledDateGrid = styled(Grid)`
-    margin-top: 32px;
-    margin-bottom: 23px;
-`
 
-const StyledFooter = styled(Grid)`
-    min-width: 25vw;
-`
 const StyledItemsFooter = styled(Grid)`
     margin-top: 16px;
-    min-width: 25vw;
+`
+
+const StyledSelect = styled(TextField)`
+    background-color: #8da57c;
+    border-radius: 4px;
+    margin-top: 4px;
 `
 
 const DmEntryCreateForm = ({
     type,
-    onCloseDrawer = () => {},
+    onCloseDrawer,
     editData,
     editId = 0,
     user_id,
```

Comment:
is this `user_id` used anywhere?

Reply:
I think it's being passed to backend through the request.